### DISCUSSION TITLE
Building a Backend with Echo Framework: Redis vs PostgreSQL?

### DISCUSSION THREAD

**Discussion by aixart12** [gosom/google-maps-scraper#79] [Q&A]
I'm currently working on transitioning a command-line tool into a backend using the Echo framework in Go. I’m also in the process of learning Go and building a frontend for this tool.

Here's my main question: if I don't need to save any persistent data, would it make more sense to use Redis instead of PostgreSQL? I’m leaning towards Redis for caching and in-memory operations, but I’d love to hear opinions on the best approach.

Any advice on structuring this project or optimizing it for performance would be much appreciated!

**Comment by gosom** [comment]
@aixart12 thanks for the interest in building a frontend tool. 
I already have something in place that I plan to push today.

**Reply by aixart12** [reply]
@gosom   Thanks for the update! That sounds great. I'm looking forward to seeing what you've built. Once you push the changes, feel free to let me know if there’s anything specific I can help.

**Reply by gosom** [reply]
@aixart12 version 1.4.0 is released. It would be nice if you could check functionality.

**Reply by aixart12** [reply]
Sure , I will look into it and thanks for update.
